simple-storage/src/main/java/com/sromku/simple/storage/ExternalStorage.java

```diff
@@ -16,6 +16,8 @@
  */
 public class ExternalStorage extends AbstractDiskStorage {
 
+	private String publicDirectory = null;
```

Comment:
please use the Android convention way of declaring private members. use `m` as for prefix. In this example it would be: `mPublicDirectory`